assets/css/style.css

```diff
@@ -1609,6 +1581,63 @@ margin-top: 5px;
   padding-left:5px;
 
 }
+@media (min-width: 768px) {
```

Comment:
At mobile size, the IG images hang over the edge a bit. Maybe it should be two columns or move the title above the images.

Reply:
I figure you meant ipad -- I just updated the grid to a new look so that the text is on top. I also centered it because it felt a bit off. Since I centered it for ipad I have it for desktop to. Let me know what you think of these changes, I can always revert back if we don't like the look